### DISCUSSION TITLE
Something for the readME?

### DISCUSSION THREAD

**Discussion by bjdaly** [versat/cntlm#74] [Ideas]
It doesn't feel like an Issue as such as people should work this out for themselves however, and forgive me if this is not the place for this as I am new to engaging on GitHub, I recently got stuck installing cntlm on macOS Monterey - the problem was that macOS Monterey blocks access to the traditional /usr/ bin ... sbin ... share and so on  - need to use /usr/local/ - whilst the readME is clear on how to change the directories, perhaps add a note in the readMe specifically for macOS Monterey users?

**Comment by fralken** [comment]
Hello, maybe rather than updating the readme we could modify the makefiles to install under /usr/local instead of /usr, at least for macOs platform.

**Reply by jschwartzenberg** [reply]
The default prefix should always be `/usr/local`. When packaging this should be overridden to `/usr`.

**Comment by fralken** [comment]
I never tried "make install" before, but on Monterey the "-d" option of the install command doesn't work, I had to delete it. @bjdaly is it the same for you? and yes, it is possible to install in /usr/local/sbin and /usr/local/share.
These folders could become the default also for the other platforms, not only darwin.
